Example/BonMot/Cells/TagStylesCell.m

```diff
@@ -0,0 +1,40 @@
+//
+//  TagStylesCell.m
+//  BonMot
+//
+//  Created by Nora Trapp on 3/3/16.
+//  Copyright © 2016 Zev Eisenberg. All rights reserved.
+//
+
+#import "TagStylesCell.h"
+
+@interface TagStylesCell ()
+
+@property (weak, nonatomic) IBOutlet UILabel *label;
+
+@end
+
+@implementation TagStylesCell
+
++ (NSString *)title
+{
+    return @"Tag Styles";
+}
+
+- (void)awakeFromNib
+{
+    [super awakeFromNib];
+
+    BONChain *boldChain = BONChain.new.fontNameAndSize(@"Baskerville-Bold", 15);
+    BONChain *italicChain = BONChain.new.fontNameAndSize(@"Baskerville-Italic", 15);
+
+    BONChain *baseChain = BONChain.new.fontNameAndSize(@"Baskerville", 17)
+                              .tagStyles(@[ BONTagMake(@"bold", boldChain), BONTagMake(@"italic", italicChain) ])
```

Comment:
Ooops, forgot to update this one when I switched the default to take a dictionary.